src/libraries/System.Text.Json/tests/System.Text.Json.SourceGeneration.Tests/NETStandardContextTests.cs

```diff
@@ -0,0 +1,25 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text.Json.Serialization;
+using System.Text.Json.Serialization.Metadata;
+using Xunit;
+
+namespace System.Text.Json.SourceGeneration.Tests.NETStandard
+{
+    public class NETStandardContextTests
+    {
+        [Fact]
+        public void RoundTripMyPoco()
```

Comment:
It may make sense to add a `summary` comment to this test, quickly explaining what it is testing.